### PR TITLE
[IIGO ]Term length is now 4 by config, exposing term length to clients

### PR DESCRIPTION
**Pull request by Spruudis**
You know that scene at the start of the mummy where the guy goes picks up a gold bug and then bugs start crawling all over him and under his skin. Yeah

### CODE REVIEW

**Comment by QFSW** [comment]
# ⚠️Warning! This diff contains changes in protected folders

File: params.go [hunk 1]
```diff
@@ -328,6 +328,24 @@ var (
 		"Sanction length for all sanctions",
 	)
 
+	iigoTermLenghtPresident = flag.Uint(
```
Comment (Neelesh99) [inline params.go]: Mispelled length here.

File: params.go [hunk 2]
```diff
@@ -328,6 +328,24 @@ var (
...
+		"iigoTermLenghtPresident",
+		4,
+		"Length of the term for the President",
+	)
+
+	iigoTermLenghtSpeaker = flag.Uint(
```
Comment (Neelesh99) [inline params.go]: As above

File: params.go [hunk 3]
```diff
@@ -328,6 +328,24 @@ var (
...
+		"iigoTermLenghtSpeaker",
+		4,
+		"Length of the term for the Speaker",
+	)
+
+	iigoTermLenghtJudge = flag.Uint(
```
Comment (Neelesh99) [inline params.go]: As above

**Review by Neelesh99 — APPROVED** [review]
Pre-approving, pls fix typos

File: params.go (hunk 1 above)
Comment (Spruudis) [reply params.go]: I should have caught that, I misspell length all the time :D

**Pull request merged**